test/integration/render.test.js

```diff
@@ -264,4 +278,61 @@ describe('render', function () {
       );
     });
   });
+
+  // Skip Link
+  // ---------------------------------------------------------------------------
+  describe('skip link', () => {
+    test('renders default skip link and label', async () => {
+      await docsifyInit();
+
+      const elm = document.getElementById('skip-to-content');
+      const expectText = 'Skip to main content';
+
+      expect(elm.textContent).toBe(expectText);
+      expect(elm.outerHTML).toMatchInlineSnapshot(
+        `"<button id=\\"skip-to-content\\">Skip to main content</button>"`
+      );
+    });
+
+    test('renders custom label from config string', async () => {
+      const expectText = 'test';
+
+      await docsifyInit({
+        config: {
+          skipLink: expectText,
+        },
+      });
+
+      const elm = document.getElementById('skip-to-content');
+
+      expect(elm.textContent).toBe(expectText);
+    });
+
+    test('renders custom label from config object', async () => {
+      const expectText = 'test';
+
+      await docsifyInit({
+        config: {
+          skipLink: {
+            '/': expectText,
```

Comment:
Would it make be good to also test a path like `/es/`?

Also what if the site has no README, and the default landing is something like `/foo` and `/es/foo`?

Reply:
Yep. Wil do.

Reply:
Done.